include/fmt/chrono.h

```diff
@@ -793,11 +793,7 @@ struct chrono_formatter {
 
   explicit chrono_formatter(FormatContext& ctx, OutputIt o,
                             std::chrono::duration<Rep, Period> d)
-      : context(ctx), out(o), val(d.count()), negative(false) {
-    if (d.count() < 0) {
-      val = 0 - val;
-      negative = true;
-    }
+      : context(ctx), out(o), val(static_cast<rep>(std::abs(d.count()))), negative(d.count() < 0) {
```

Comment:
I suggest keeping the old code where negative case is handled in one place.

Reply:
You are trying to initialize 'val' which is unsigned with a signed value (d.count()).

Reply:
Adding a cast is fine. I'm just saying that having negation and setting `negative` in one if block makes the intent a bit clearer, so let's keep that part.

Reply:
OK